src/js/common/schemaform/SaveFormLink.jsx

```diff
@@ -63,9 +72,18 @@ class SaveFormLink extends React.Component {
             user={this.props.user}
             onUpdateLoginUrl={this.props.onUpdateLoginUrl}
             onLogin={this.saveFormAfterLogin}/>
-        {savedStatus === SAVE_STATUSES.failure &&
-          <div className="usa-alert usa-alert-error no-background-image schemaform-save-error">We’re sorry, but something went wrong. Please try saving your application again.</div>}
-        {content}
+        {saveErrors.has(savedStatus) &&
+          <div role="alert" className="usa-alert usa-alert-error no-background-image schemaform-save-error">
```

Comment:
Is it now common practice for alerts to not include the background image (was this an exclamation mark before)? It seems appropriate here?